src/gcodePlanner.cpp

```diff
@@ -409,15 +409,23 @@ void GCodePlanner::addPolygonsByOptimizer(Polygons& polygons, GCodePathConfig* c
     {
         return;
     }
-    PathOrderOptimizer orderOptimizer(lastPosition, z_seam_pos, z_seam_type);
-    for (unsigned int poly_idx = 0; poly_idx < polygons.size(); poly_idx++)
+    if (spiralize)
     {
-        orderOptimizer.addPolygon(polygons[poly_idx]);
+        // only interested in the outer perimeter
+        addPolygon(polygons[0], 0, config, wall_overlap_computation, wall_0_wipe_dist, spiralize);
```

Comment:
As you wish but that leaves spiralize completely broken as it will output both the outer and inner polygons which crash into each other for models such as I have above.